api/base/filters.py

```diff
@@ -215,6 +216,8 @@ def convert_value(self, value, field):
                 )
         elif isinstance(field, self.LIST_FIELDS):
             return value
+        elif isinstance(field, (RelationshipField, TargetField)):
```

Comment:
I don't think `RelationshipField` and `TargetField` belong in `self.LIST_FIELDS`. But I do agree the comparison can be combined since they all return `value`. I can add `self.RELATIONSHIP_FIELDS = (RelationshipField, TargetField)` to follow the patten in this mixin.